cg/services/file_delivery/file_formatter_service/utils/utils.py

```diff
@@ -0,0 +1,17 @@
+import os
+from pathlib import Path
+
+from cg.constants.delivery import INBOX_NAME
+
+
+def is_inbox_path(file_path: Path):
+    return INBOX_NAME in str(file_path)
+
+
+def get_ticket_dir_path(file_path) -> Path:
```

Comment:
As mentioned this looks quite flimsy to me. What if the incoming path is /home/something/cust000/inbox/789789/case/sample. Then we would get something else than if it was just /home/something/cust000/inbox/789789/case

Reply:
As explained above, at this stage the path will be /home/something/cust000/inbox/789789/ for all files, after the formatting files are placed into their respective case and sample folders.

Reply:
Resolving this in favour of the discussion here: https://github.com/Clinical-Genomics/cg/pull/3626#discussion_r1734170026